.travis.yml

```
@@ -1,4 +1,5 @@
 ---
+github_token=$NOT_KYTRINYX_TOKEN
```

I don't know much about Travis, but should this definition be part of an `env` key: https://docs.travis-ci.com/user/environment-variables/#defining-public-variables-in-travisyml

---

The configlet script uses uppercase:
```suggestion
GITHUB_TOKEN=$NOT_KYTRINYX_TOKEN
```